pkg/webhook/resources/setting/validator.go

```diff
@@ -162,6 +166,14 @@ func (v *settingValidator) validateBackupTarget(setting *v1beta1.Setting) error
 		return werror.NewInvalidError("NFS backup target should not have bucket name or region ", "value")
 	}
 
+	vmBackups, err := v.vmBackupCache.List(metav1.NamespaceAll, labels.Everything())
```

Comment:
Is it possible to query specific namespaces only?

Reply:
Probably no. VMBackup depends on VM in which namespace, so it's better to check all namespace.